uwblueprint/paramedics-react
# Patient delete button

## Overview
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

[Notion task](https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=e0152f69c6d1415f8892686284db86db)


## Changes
- Created delete button according to designs
- Added GraphQL mutation for deleting patients for frontend

## Testing
- Navigate to a particular patient in your frontend (`/events/eventId/ccps/ccpId/patients/patientId`)
- Ensure that the delete patient button is present and the confirm modal is working as expected
- When you click 'delete', check the backend and determine whether the patient in question has a `status: Deleted`
- Make sure that the CCP dashboard is updated with the new deleted status when you navigate back
## Screenshots
![Kapture 2020-09-03 at 13 51 26](https://user-images.githubusercontent.com/21224282/92160705-ad132800-edec-11ea-8268-439da9325161.gif)
